metabase/metabase
# Render parameter type and operator in parameter sidebar

Closes https://github.com/metabase/metabase/issues/41168

### Description

This PR renders parameter type and operator in the parameter sidebar in readonly mode

[design approval](https://metaboat.slack.com/archives/C0645JP1W81/p1712947133812129?thread_ts=1712868558.200839&cid=C0645JP1W81)

### How to verify

- create a dashboard with question
- add new parameter
- click on parameter - sidebar should show parameter name and operator 

### Demo


<img width="1126" alt="Screenshot 2024-04-11 at 23 29 48" src="https://github.com/metabase/metabase/assets/125459446/d1f83cb6-70c8-448a-b07b-e0a1baf19de7">


https://github.com/metabase/metabase/assets/125459446/fa40d20b-e607-428e-838b-9157c5dece64


### Checklist
